src/main/resources/application.yaml

```diff
@@ -50,4 +50,8 @@ acas:
 tornado:
   api:
     url: ${TORNADO_URL:http://localhost:8090/rs/render}
-    key: ${TORNADO_ACCESS_KEY:ignored_default_value}
\ No newline at end of file
+    key: ${TORNADO_ACCESS_KEY:ignored_default_value}
+
+notifications:
+  govNotifyApiKey: ${gov-notify-api-key:}
+  sampleEmailTemplateId: "4ea3cf16-0cbb-4f27-8cfb-a084b126086e"
```

Comment:
Is this where we could extend for other templates by adding new template ID's here?